lib/auth/middleware.js

```diff
@@ -0,0 +1,35 @@
+export function validArtsyEmail(email) {
+  return /@artsymail.com$/.test(email) || /@artsy.net$/.test(email);
+}
+
+export function isUserAdmin(user) {
+  return user.type === 'Admin' || user.roles.indexOf('admin') !== -1;
+}
+
+export function isAuthable(req) {
+  return req.accepts(['json', 'html']) === 'html';
+}
+
+export function authenticateWithUser(req) {
```

Comment:
I back this, in prod you must be an admin, any other environment you can be any Artsy user.